data/json/recipes/armor/storage.json

```diff
@@ -256,7 +247,7 @@
     "qualities": [ { "id": "SEW", "level": 1 } ],
     "components": [
       [ [ "leather", 4 ], [ "rag", 4 ] ],
-      [ [ "string_6", 3 ], [ "thread", 10 ], [ "sinew", 10 ], [ "plant_fibre", 10 ], [ "yarn", 10 ] ]
+      [ [ "string_6", 3 ], [ "withered", 6 ], [ "straw_pile", 6 ], [ "thread", 10 ], [ "sinew", 10 ], [ "plant_fibre", 10 ], [ "yarn", 10 ] ]
```

Comment:
Filament could potentially be bundled into the `cordage_small` requirement, though making the thread usage equal to the amount of thread needed to make a string might be desirable.